src/main/app/forms/models/address.ts

```diff
@@ -25,21 +26,46 @@ export class ValidationConstants {
 
 export class Address implements CompletableTask {
 
+  @ValidateIf(o => o.addressVisible)
   @IsDefined({ message: ValidationErrors.FIRST_LINE_REQUIRED, groups: ['claimant', 'defendant', 'response'] })
   @IsNotBlank({ message: ValidationErrors.FIRST_LINE_REQUIRED, groups: ['claimant', 'defendant', 'response'] })
-  @MaxLength(ValidationConstants.ADDRESS_MAX_LENGTH, { message: ValidationErrors.FIRST_LINE_TOO_LONG, groups: ['claimant', 'defendant', 'response'] })
+  @MaxLength(ValidationConstants.ADDRESS_MAX_LENGTH, {
+    message: ValidationErrors.FIRST_LINE_TOO_LONG,
```

Comment:
why not `(global)ValidationErrors.TEXT_TOO_LONG`

Reply:
didn't change that